proto/controlclient.proto

```diff
@@ -67,10 +68,18 @@ message _SigningKey {
 }
 
 message _ListSigningKeysRequest {
-  string next_token = 1; 
+  string next_token = 1;
 }
 
 message _ListSigningKeysResponse {
   repeated _SigningKey signing_key = 1;
   string next_token = 2;
 }
+
+message _FlushCacheRequest {
+  string cache_name = 1;
```

Comment:
do we need cache name here?

Reply:
nm I was getting confused with data plane APIs where we are sending that as a header.